src/go/configgenerator/listener_generator.go

```diff
@@ -639,27 +639,16 @@ func makeTranscoderFilter(serviceInfo *sc.ServiceInfo) *hcmpb.HttpFilter {
 				DescriptorSet: &transcoderpb.GrpcJsonTranscoder_ProtoDescriptorBin{
 					ProtoDescriptorBin: configContent,
 				},
-				AutoMapping:       true,
-				ConvertGrpcStatus: true,
+				AutoMapping:                  true,
+				ConvertGrpcStatus:            true,
+				IgnoredQueryParameters:       transcodingIgnoreQueryParameters(serviceInfo),
+				IgnoreUnknownQueryParameters: serviceInfo.Options.TranscodingIgnoreUnknownQueryParameters,
```

Comment:
If IgnoreUnknownQueryParameters is true,  we don't need to set another one.  It doesn't make sense to set both.

Reply:
Good point, will add exception in start_esp.py